sdk/core/core-http/review/core-http.api.md

```diff
@@ -7,11 +7,13 @@
 import { AbortSignalLike } from '@azure/abort-controller';
 import { AccessToken } from '@azure/core-auth';
 import { Context } from '@azure/core-tracing';
+import { createSpanFunction } from '@azure/core-tracing';
 import { Debugger } from '@azure/logger';
 import { GetTokenOptions } from '@azure/core-auth';
 import { isTokenCredential } from '@azure/core-auth';
 import { OperationTracingOptions } from '@azure/core-tracing';
 import { Span } from '@azure/core-tracing';
+import { CreateSpanFunctionArgs as SpanConfig } from '@azure/core-tracing';
```

Comment:
These used to be named SpanConfig. Richard made them compatible with `CreateSpanFunctionArgs` so I was thinking exporting both in case someone is using the named type?

Reply:
I'm less fussed about losing interface types for arguments, but as a sop to compat it could still be aliased as deprecated I suppose.